plugins/image-source/obs-slideshow.c

```diff
@@ -179,6 +223,23 @@ static bool valid_extension(const char *ext)
 	       astrcmpi(ext, ".gif") == 0;
 }
 
+static void do_transition(void *data, bool to_null)
+{
+	struct slideshow *ss = data;
+
+	if (ss->use_cut) {
+		obs_transition_set(ss->transition,
+				ss->files.array[ss->cur_item].source);
+	} else if (!to_null && !ss->use_cut) {
+		obs_transition_start(ss->transition,
+				OBS_TRANSITION_MODE_AUTO,
+				ss->tr_speed,
+				ss->files.array[ss->cur_item].source);
+	} else if (to_null && !ss->use_cut) {
+		obs_transition_set(ss->transition, NULL);
+	}
```

Comment:
Style:  Curly braces are not needed for this `if` statement, since the contents of each branch is a single statement.